client/orm/orm.go

```diff
@@ -644,6 +643,8 @@ func NewOrmWithDB(driverName, aliasName string, db *sql.DB, params ...DBOption)
 }
 
 func newDBWithAlias(al *alias) Ormer {
+	BootStrapWithAlias(al.Name) // execute only once
```

Comment:
why call this function here? I mean, you can:
```go
NewDBXXX()
// manually
BootStrapWithAlias()
```

Reply:
the reason i put it under `newDBWithAlias` was that we have to call `BootStrap` in both the default and non-default DB.